src/Components/appointments/Appointment.js

```diff
@@ -1,20 +1,37 @@
 import React, { useState, useEffect } from "react";
```

Comment:
You use an snackbar only for an error and with the same message, so you don't need states for the text with the status and write directly.